tests/data/foo.1234.js

```diff
@@ -1,2 +1,2 @@
-require=function r(n,o,e){function t(f,i){if(!o[f]){if(!n[f]){var c="function"==typeof require&&require;if(!i&&c)return c(f,!0);if(u)return u(f,!0);var a=new Error("Cannot find module '"+f+"'");throw a.code="MODULE_NOT_FOUND",a}var p=o[f]={exports:{}};n[f][0].call(p.exports,function(r){var o=n[f][1][r];return t(o?o:r)},p,p.exports,r,n,o,e)}return o[f].exports}for(var u="function"==typeof require&&require,f=0;f<e.length;f++)t(e[f]);return t}({1:[function(r,n){function o(r){return"bar"+r}n.exports=o},{}],2:[function(r,n){var o=r("./bar"),e=function(r){return o(r)+o(r)};n.exports=e},{"./bar":1}],foo:[function(r,n){n.exports=r("./foo")},{"./foo":2}]},{},[]);
-//# sourceMappingURL=foo.1234.js.map
+!function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){return o(e[i][1][r]||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}({1:[function(require,module,exports){"use strict";module.exports=function(x){return"bar".concat(x)}},{}],2:[function(require,module,exports){"use strict";var bar=require("./bar");module.exports=function(x){return bar(x)+bar(x)}},{"./bar":1}]},{},[2]);
+//# sourceMappingURL=foo
```

Comment:
Is it supposed to be just `foo`?

Reply:
No, that's weird. It's not covered in the tests, but I'll fix that. I'm not sure why terser is removing the end of the URL.